src/index.js

```diff
@@ -354,17 +344,30 @@ function ncc (
             loader: eval('__dirname + "/loaders/uncacheable.js"')
           },
           {
-            loader: eval('__dirname + "/loaders/ts-loader.js"'),
+            loader: eval('__dirname + "/loaders/swc-loader.js"'),
             options: {
-              transpileOnly,
-              compiler: eval('__dirname + "/typescript.js"'),
-              compilerOptions: {
-                module: 'esnext',
-                target: 'esnext',
-                ...fullTsconfig.compilerOptions,
-                allowSyntheticDefaultImports: true,
-                noEmit: false,
-                outDir: '//'
+              minify: false, // TODO: maybe we could omit terser if `true`?
```

Comment:
```suggestion
              minify: false,
```

The `TODO` here should be removed. The minify shouldn't be done during bundling (otherwise the runtime code introduced by webpack can not be minimized). IMHO, if swc's minify should be used, the `TerserWebpackPlugin` should be used instead.

Reply:
The latest commit resolves the problem. The review can be marked as resolved now.